src/models/image-map.ts

```diff
@@ -92,6 +97,13 @@ export const ImageMapModel = types
     },
     getCachedImage(url?: string) {
       return url ? self.images.get(url) : undefined;
+    },
+    getCurriculumUrl() {
```

Comment:
```suggestion
    get curriculumUrl() {
```

With that change then you can just do `imageMap.curriculumUrl`.
It also improves the performance because MST will cache the value until something changes. But perhaps you tried it and ran into problems.